### ISSUE TITLE
[MU4 Issue] [Accessibility] Voice Over doesn't read almost all elements (Mac)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9153]
**Describe the bug**
Voice Over doesn't read almost all elements (Mac)

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: Mac10.15.6

**Additional context**

**Comment by DmitryArefiev** [comment]
Checked #9421/#9153 on Win10, **Mac11**, LinuxMint 20 - FIXED
